src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -253,6 +255,14 @@ private static String createQuery(Map<String, Object> params)
 							nestedEntry.getValue());
 				}
 				flatParams.putAll(flattenParams(flatNestedMap));
+			} else if (value instanceof List<?>) {
+				Map<String, Object> flatNestedMap = new LinkedHashMap<String, Object>();
+				int index = 0;
+				for (Object element : (List<?>) value) {
```

Comment:
This may be cleaner. It also doesn't allocate a new object for each iteration of the loop:

```java
Iterator<String> myIter = value.iterator();
for (int index=0; myIter.hasNext(); ++index) {
 flatNestedMap.put(String.format("%s[%s], key, index), myIter.next());
}
```

Both are correct, so feel free to #ignore if you don't want to.